Lesson2/src/Homework.java

```diff
@@ -0,0 +1,95 @@
+import java.util.Scanner;
+
+public class Homework {
+
+    public static final long integerMaxValue = Integer.MAX_VALUE;
+
+    public static void main(String[] args) {
+//        System.out.println(sum(2147483647, 20));
+//        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
+//        System.out.println(max(50, 49));
+//        System.out.println(average(new int[]{1, 2, 3, 4, 5}));
+//        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+//        System.out.println(calculateHypotenuse());
+    }
+
+    /**
+     * 1. Method should return sum of "a" and "b"
+     * 2. (optional) if sum of a and b is more then int can store -- return -1;
+     **/
+    public static int sum(int a, int b) {
+        return ((long) a + (long) b) > integerMaxValue ? (-1) : (a + b);
```

Comment:
the solution is good, you can remove some extra parentheses though, like
``
return (long) a + (long) b > Integer.MAX_VALUE ? -1 : a + b;
``

Reply:
Ok, thanks. Just tried to make re-request to reviewer.